lib/CommandImplementations.ts

```diff
@@ -195,7 +208,12 @@ export async function handleGen3Count(msg: Message, args: string): Promise<void>
     }
 
     if (address !== '') {
-        replyWithMention(msg, `You are currently set to receive ${gen3Count} generation 3 slug${gen3Count > 1 ? 's' : ''}! (${burns} eligible burns)`);
+        if (gen3Count < 1) {
```

Comment:
```suggestion
        if (gen3Count === 0) {
```

I think it would be a little clearer to compare to zero here